pandas/core/sparse/array.py

```diff
@@ -614,6 +614,24 @@ def fillna(self, value, downcast=None):
         return self._simple_new(new_values, self.sp_index,
                                 fill_value=fill_value)
 
+    def all(self, axis=0, *args, **kwargs):
+        """
+        Tests whether all elements evaluate True
+
+        Returns
+        -------
+        all : bool
+        """
+        nv.validate_all(args, kwargs)
+
+        values = self.sp_values
+
+        if len(values) != len(self):
+            values = values.tolist()
```

Comment:
is there a reason you are converting to a list? and not a dense array?

Reply:
@jreback For the performance reason. This is sparse and the converting to dense is not efficient.

Reply:
I am asking why you are converting ``.tolist()``, then appending, that is non-performant.

Reply:
Simply I wanted to use `np.all`. I got an idea not to use `tolist()`. I'll change the solution.